lib/content-services/src/lib/permission-manager/components/permission-container/permission-container.component.scss

```diff
@@ -6,6 +6,10 @@
     &.adf-datatable {
         overflow: hidden;
 
+        .adf-datatable-row .adf-permission-role-column-header {
+            position: relative;
```

Comment:
I see you moved this style from permission-list.component to this component. Can you please confirm that we do not need the height property as well?

Reply:
height is not needed, since we haven't removed it from where it originally was, it is not making any difference.

Reply:
Got it. Thanks!!